.drone.yml

```
@@ -16,7 +16,7 @@ environment:
 
 services:
   - name: keycloak
-    image: jboss/keycloak
+    image: quay.io/keycloak/keycloak:15.1.1
```

```suggestion
    image: quay.io/keycloak/keycloak:15.0.2
```

Just to ensure we're running exactly whats in prod.